.github/workflows/prebuild.yml

```diff
@@ -79,3 +82,40 @@ jobs:
       - uses: ./.github/actions/environment
       - name: "Check release commit message format"
         run: ./ci/check_release_commit_messages.sh
+
+  # Warn if core/embed/rust/Cargo.lock changed
+  cargolock_check:
+    name: Cargo.lock check
+    if: ${{ github.ref != 'main' && github.event_name == 'pull_request' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Ask git
+        id: git_diff
+        run: |
```

Comment:
We can maybe simplify it to:
```
run: |
    git diff --exit-code origin/main -- core/embed/rust/Cargo.lock || echo "cargo_modified=1" >> $GITHUB_ENV
```

This should create `env.cargo_modified` variable which can be used in subsequent steps as:
`if: ${{ env.cargo_modified == '1' }}`

WDYT

Reply:
Sounds good, didn't realize these persist between jobs.